addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -142,29 +142,31 @@ export class A11YPanel extends Component<A11YPanelProps, A11YPanelState> {
 
     return active ? (
       <Fragment>
-        <ScrollArea vertical horizontal>
-          <Tabs
-            key="tabs"
-            tabs={[
-              {
-                label: <Violations>{violations.length} Violations</Violations>,
-                panel: (
-                  <Report passes={false} items={violations} empty="No a11y violations found." />
-                ),
-              },
-              {
-                label: <Passes>{passes.length} Passes</Passes>,
-                panel: <Report passes items={passes} empty="No a11y check passed." />,
-              },
-              {
-                label: <Incomplete>{incomplete.length} Incomplete</Incomplete>,
-                panel: (
-                  <Report passes={false} items={incomplete} empty="No a11y incomplete found." />
-                ),
-              },
-            ]}
-          />
-        </ScrollArea>
+        {status === 'running' ? null : (
```

Comment:
Nit-picking but maybe we can show a loading spinner here?

Reply:
Done :)